docs/adr/adr-003-march2022-testnet.md

```diff
@@ -0,0 +1,159 @@
+# ADR #003: March 2022 Testnet Celestia Node
+
+<hr style="border:3px solid gray"> </hr>
+
+## Authors
+
+@renaynay @Wondertan
+
+## Changelog
+
+* 2021-11-25: initial draft
+
+<hr style="border:2px solid gray"> </hr>
+
+## Legend
+
+### Celestia DA Network
+
+Refers to the data availability "halo" network created around the Core network.
+
+### **Bridge Node**
+
+A **bridge** node is a **full** node that is connected to a Celestia Core node via RPC. It receives either a remote
+address from a running Core node or it can run a Core node as an embedded process, but the critical difference is that
+instead of constructing blocks via sampling the network for shares, it receives headers and blocks directly from its 
```

Comment:
it still uses the process of sampling to download the entire block, it will just be systematic rather than random correct? I think using the term sampling is still fine, since you can sample to 1. prove data is available and 2. reconstruct the entire block.